sensor/kubernetes/certrefresh/certificates/requester.go

```diff
@@ -53,61 +57,68 @@ func NewSecuredClusterCertificateRequester(sendC chan<- *message.ExpiringMessage
 	](
 		sendC,
 		receiveC,
-		&SecuredClusterMessageFactory{},
-		&SecuredClusterResponseFactory{},
+		&securedClusterMessageFactory{},
+		&securedClusterResponseFactory{},
+		func() *centralsensor.CentralCapability {
+			centralCap := centralsensor.CentralCapability(centralsensor.SecuredClusterCertificatesReissue)
+			return &centralCap
+		}(),
 	)
 }
 
-func newRequester[ReqT any, ResT ProtobufResponse](
+func newRequester[ReqT any, RespT protobufResponse](
 	sendC chan<- *message.ExpiringMessage,
-	receiveC <-chan ResT,
-	messageFactory MessageFactory,
-	responseFactory ResponseFactory[ResT],
-) *GenericRequester[ReqT, ResT] {
-	return &GenericRequester[ReqT, ResT]{
-		sendC:           sendC,
-		receiveC:        receiveC,
-		messageFactory:  messageFactory,
-		responseFactory: responseFactory,
+	receiveC <-chan RespT,
+	messageFactory messageFactory,
+	responseFactory responseFactory[RespT],
+	centralCapability *centralsensor.CentralCapability,
+) *genericRequester[ReqT, RespT] {
+	return &genericRequester[ReqT, RespT]{
+		sendC:             sendC,
+		receiveC:          receiveC,
+		messageFactory:    messageFactory,
+		responseFactory:   responseFactory,
+		centralCapability: centralCapability,
 	}
 }
 
-type GenericRequester[ReqT any, ResT ProtobufResponse] struct {
-	sendC           chan<- *message.ExpiringMessage
-	receiveC        <-chan ResT
-	stopC           concurrency.ErrorSignal
-	requests        sync.Map
-	messageFactory  MessageFactory
-	responseFactory ResponseFactory[ResT]
+type genericRequester[ReqT any, RespT protobufResponse] struct {
+	sendC             chan<- *message.ExpiringMessage
+	receiveC          <-chan RespT
+	stopC             concurrency.ErrorSignal
+	requests          sync.Map
+	messageFactory    messageFactory
+	responseFactory   responseFactory[RespT]
+	centralCapability *centralsensor.CentralCapability
```

Comment:
Maybe call this
```suggestion
	requiredCentralCapability *centralsensor.CentralCapability
```
?

Reply:
That's actually how I wanted to name it originally, will do.